README.md

```diff
@@ -4,20 +4,55 @@
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 [![codecov](https://codecov.io/gh/BlockStoreApp/Web3Swift/branch/develop/graph/badge.svg?token=SY7mpMQbGs)](https://codecov.io/gh/BlockStoreApp/Web3Swift)
 
-## Example
-
-To run the example project, clone the repo, and run `pod install` from the Example directory first.
-
 ## Installation
 
-Web3Swift is not yet available through [CocoaPods](http://cocoapods.org).
-<!-- To install
-it, simply add the following line to your Podfile:
+Web3Swift is available through [CocoaPods](http://cocoapods.org).
+To install it, simply add the following line to your Podfile:
 
 ```ruby
-pod 'Web3Swift'
+pod 'Web3Swift', :git => 'https://github.com/BlockStoreApp/Web3Swift.git'
+```
+
+## Sending ether
+
+To send some wei from an account with a private key `0x1636e10756e62baabddd4364010444205f1216bdb1644ff8f776f6e2982aa9f5` to an account with an address `0x79d2c50Ba0cA4a2C6F8D65eBa1358bEfc1cFD403` on a mainnet:
```

Comment:
Let's use `0x0000000000000000000000000000000000000000` account as receipient? 
And add a public address to private key.